app/containers/message/Components/RightIcons/ReadReceipt.tsx

```diff
@@ -6,15 +6,13 @@ import { useTheme } from '../../../../theme';
 
 const ReadReceipt = React.memo(({ isReadReceiptEnabled, unread }: { isReadReceiptEnabled?: boolean; unread?: boolean }) => {
 	const { colors } = useTheme();
+	const iconName = unread ? 'check' : 'check-double';
+	const iconColor = unread ? colors.fontAnnotation : colors.fontInfo;
+	const size = unread ? 20 : 25;
+	const marginTop = unread ? { marginTop: -3 } : { marginTop: -5 };
```

Comment:
Doesn't look good.
Does it work properly on 2x?
Different screen sizes on Android?